RuleDocumentation/AvoidUsingPlainTextForPassword.md

```diff
@@ -1,31 +1,33 @@
 # AvoidUsingPlainTextForPassword
 
-**Severity Level: Warning**
+--Severity Level: Warning--
```

Comment:
I don't think `--` renders differently than literally, the `**` is intended for bold

Reply:
Not sure how that change happened. Reverted to bold.